src/utils/index.js

```diff
@@ -140,5 +142,11 @@ export const initialSignupCredState = {
   confirmPassword: "",
   name: "",
 };
-
+export const initialPostState = {
+  content: "",
+  mediaURLs: [],
+};
+export const notify = (content, type = "success") => toast(content, { type });
 export const initialLoginCredState = { email: "", password: "" };
+export { callApi };
+export const API_URL = "https://zengram.herokuapp.com/api";
```

Comment:
Suggestion:You can store it in an .env file

Reply:
okay